### ISSUE TITLE
Not using GPT-4

### ISSUE THREAD

**Issue by BenGoAway** [deiucanta/chatpad#81]
Hi, loaded the docker of ChatPad rather like it. 

GPT-3.5 Turbo does fine works perfectly but GPT-4 just won't. 

I have the setting at GPT-4 (Limited Beta) but I did try a few of the others. The API key should have access other things I have do. 

Did I miss something?

**Comment by BenGoAway** [comment]
resolved issue, not entirely sure what was happening but I deleted the container and reloaded it and its been making requests to GPT 4.

**Comment by trentnbauer** [comment]
How are you testing if you're using GPT4?
I have mine set to GPT4 per below, 
![image](https://github.com/deiucanta/chatpad/assets/45578026/06e67158-90a9-4b7f-bd22-4f7522d5f231)
But asking what model it is, it gives gpt3
![image](https://github.com/deiucanta/chatpad/assets/45578026/807ce1f3-515c-4cc3-adc3-aa9f145fe97a)

Deleting and re-creating the container didn't help for me

**Comment by vicenterusso** [comment]
I believe the API is not aware of the model being used. Making direct requests to OpenAI yields the same results.

**Comment by Supernova3339** [comment]
> I believe the API is not aware of the model being used. Making direct requests to OpenAI yields the same results.

That is correct. If you define it in it's system prompt it will return it, else not.

**Comment by mpolyntsov** [comment]
How to use GPT4 then?

**Comment by Necka44** [comment]
I have the same issue running the desktop app. I selected GPT-4-1106-Preview, open up a chat, select GPT4 (here, often it resets the model to GPT4 (limited beta).
![image](https://github.com/deiucanta/chatpad/assets/39954105/58715c20-c69b-4b1a-a776-278251d5b3c1)

If I simply ask "what is the latest available version of Unity Engine" it replies about the version of 2021.
I'm not asking it what model it's using as it replies 3 or 3.5 but asking questions about what it should know shows that the data is from 2021.
